modules/migration-editing-pvs-in-migplan.adoc

```diff
@@ -35,11 +39,15 @@ spec:
       name: mysql
       namespace: mysql-persistent
     selection:
-      action: copy
-      copyMethod: filesystem
-      storageClass: cephrbd <1>
-      accessMode: ReadWriteMany <2>
+      action: <copy> <1>
+      copyMethod: <filesystem> <2>
+      verify: <true> <3>
+      storageClass: <gp2> <4>
+      accessMode: <ReadWriteMany> <5>
     storageClass: cephfs
 ----
-<1> Destination storage class.
-<2> Destination access mode.
+<1> Allowed values are `move`, `copy`, and `skip`. If only one action is supported, the default value is the supported action. If multiple actions are supported, the default value is `copy`.
+<2> Allowed values are `snapshot` and `filesystem`. Default value is `filesystem`.
+<3> For file system copy method only: Set to `true` to verify the files after they are copied. Default value is `null`.
+<4> Storage class on the destination cluster. If the source cluster PV is Gluster or NFS, the default value is `cephfs` for `ReadWriteMany` or `cephrbd` for `ReadWriteOnce`, if they are available. If `cephfs` or `cephrbd` are not available or if the source cluster PV is a different type, the default is the storage class _for the same provisioner_, if it is available. If not, the default is the default storage class of the destination cluster.
```

Comment:
Storage classes available are those listed in `status.destStorageClasses`. The `name` from any listed storage class is a valid value for `storageClass` here.

Reply:
Are you saying that the `name` field will be populated after PV discovery? The user does not need to know it in advance?

Reply:
The entire status.destStorageClasses struct is populated automatically -- that's why we've put it in status, as there's nothing user-editable there. Basically the user selection for `selection.storageClass`, if it's not empty, must correspond to the "name" field from one of the destStorageClasses.

Reply:
Great. That makes things much clearer.